prototypes/ssb_mplementation/ssb.py

```diff
@@ -0,0 +1,201 @@
+import os
+os.environ["OMP_NUM_THREADS"] = "1"
+os.environ["MKL_NUM_THREADS"] = "1"
+os.environ["OPENBLAS_NUM_THREADS"] = "1"
+
+import numpy as np  # NOQA: E402
+import matplotlib.pyplot as plt  # NOQA: E402
+from matplotlib.colors import LogNorm  # NOQA: E402
+
+from libertem.udf import UDF  # NOQA: E402
+from libertem import api as lt  # NOQA: E402
+from libertem.executor.inline import InlineJobExecutor  # NOQA: E402
+
+from scipy.sparse import csr_matrix  # NOQA: E402
+
+
+def main():
+    path = r'C:\Users\lesnic\Nextcloud\Dieter\cGaN_sim_300kV\DPs\CBED_MSAP.raw'

```

Comment:
I believe, this line of code could cause a problem on other systems ;-)

Reply:
Yes, this is the first version for tests